cdap-common/src/main/java/co/cask/cdap/common/namespace/AbstractNamespaceClient.java

```diff
@@ -88,11 +88,10 @@ public void create(NamespaceMeta namespaceMeta)
 
     Id.Namespace namespace = Id.Namespace.from(namespaceMeta.getName());
     URL url = resolve(String.format("namespaces/%s", namespace.getId()));
-    HttpResponse response = execute(HttpRequest.put(url).withBody(new Gson().toJson(namespaceMeta)).build());
+    HttpResponse response = execute(HttpRequest.put(url).withBody(GSON.toJson(namespaceMeta)).build());
     String responseBody = response.getResponseBodyAsString();
     if (response.getResponseCode() == HttpURLConnection.HTTP_OK) {
-      if (responseBody != null && responseBody.equals(String.format("Namespace '%s' already exists.",
-                                                                    namespaceMeta.getName()))) {
+      if (responseBody.equals(String.format("Namespace '%s' already exists.", namespaceMeta.getName()))) {
```

Comment:
Nevermind. https://github.com/caskdata/common/blob/develop/common-http/src/main/java/co/cask/common/http/HttpResponse.java#L60-L62

Still, `HttpResponse.responseBody` is nullable, but I don't know what you get if you do `new String(null, Charsets.UTF_8);`. Seems that `HttpResponse.getResponseBodyAsString()` is bad. Should return null if responseBody is null probably.